docs/examples/promise/02_listen_to_blocks/index.js

```diff
@@ -10,13 +10,15 @@ async function main () {
   // Subscribe to the new headers on-chain. The callback is fired when new headers
   // are found, the call itself returns a promise with a subscription that can be
   // used to unsubscribe from the newHead subscription
-  const subscriptionId = await api.rpc.chain.subscribeNewHead((header) => {
-    console.log(`best #${header.blockNumber}`);
+  const unsubscribe = await api.rpc.chain.subscribeNewHead((header) => {
+    console.log(`Chain is at block: #${header.blockNumber}`);
   });
 
-  // Id for the subscription, we can cleanup and unsubscribe via
-  // `api.chain.newHead.unsubscribe(subscriptionId)`
-  console.log(`subsciptionId: ${subscriptionId}`);
+  // We can cleanup and unsubscribe using the unsubscribe() method of the API. In this example we're unsubscribing after 20s.
+  setTimeout(() => {
```

Comment:
As per README comment

Reply:
I removed the sentence here and changed it to the following in the README's:

> In this example we're calling the built-in unsubscribe() function after a timeOut of 20s to cleanup and unsubscribe from listening to updates.